src/main/java/com/microsoft/jenkins/azuread/AzureSecurityRealm.java

```diff
@@ -349,6 +350,7 @@ public boolean process(HttpServletRequest request, HttpServletResponse response,
     }
 
     @Extension
+    @Symbol("azureSecurityRealm")
```

Comment:
Is this needed?
I would expect the symbol to be this out of the box (it's driven off the class name)

```suggestion
```